check-result.js

```diff
@@ -0,0 +1,11 @@
+function checkResult(player, computer) {
```

Comment:
the RPS game is circular, so you cannot use greater than and less than here.  but if you compare a string to a string, you just need to account for every possible combo of throws from the player and the computer.